doc/dev_guide/building_from_source.rst

```diff
@@ -1,283 +1,264 @@
-               python27 py27-pip py27-setuptools py27-daemon \
-               py27-msgpack py27-yaml py27-argparse py27-six py27-gevent
+RHEL/CentOS (under version 8)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   If some Python modules are not available in a repository,
-   it is best to set up the modules by getting a tarball and
-   doing the setup with ``python setup.py`` like this:
+..  code-block:: console
 
-   .. code-block:: console
+    $ sudo yum install python-pip
+    $ sudo yum install epel-release
 
-       $ # On some machines, this initial command may be necessary:
-       $ wget https://bootstrap.pypa.io/ez_setup.py -O - | sudo python
+    $ yum install -y git gcc cmake3 make gcc-c++ zlib-devel readline-devel \
+      ncurses-devel openssl-devel libunwind-devel libicu-devel \
+      python3-yaml python3-six python3-gevent
 
-       $ # Python module for parsing YAML (pyYAML), for test suite:
-       $ # (If wget fails, check at http://pyyaml.org/wiki/PyYAML
-       $ # what the current version is.)
-       $ cd ~
-       $ wget http://pyyaml.org/download/pyyaml/PyYAML-3.10.tar.gz
-       $ tar -xzf PyYAML-3.10.tar.gz
-       $ cd PyYAML-3.10
-       $ sudo python setup.py install
+    $ git clone https://github.com/tarantool/tarantool.git --recursive
 
-   Finally, use Python ``pip`` to bring in Python packages
-   that may not be up-to-date in the distro repositories.
-   (On CentOS 7, it will be necessary to install ``pip`` first,
-   with :code:`sudo yum install epel-release` followed by
-   :code:`sudo yum install python-pip`.)
+    $ cd tarantool
 
-   .. code-block:: console
+    $ git submodule update --init --recursive
 
-       $ pip install -r \
-             https://raw.githubusercontent.com/tarantool/test-run/master/requirements.txt \
-             --user
+    $ make clean         # unnecessary, added for good luck
+    $ rm CMakeCache.txt  # unnecessary, added for good luck
 
-   This step is only necessary once, the first time you do a download.
+    $ mkdir build && cd build
 
-2. Use ``git`` to download the latest Tarantool source code from the
-   GitHub repository ``tarantool/tarantool``, branch 2.2, to a
-   local directory named ``~/tarantool``, for example:
+    $ # start initiating with build type=RelWithDebInfo
+    $ cmake3 .. -DCMAKE_BUILD_TYPE=RelWithDebInfo
 
-   .. code-block:: console
+    $ make
 
-       $ git clone --recursive https://github.com/tarantool/tarantool.git -b 2.2 ~/tarantool
+    $ make test
 
-   On rare occasions, the submodules need to be updated again with the
-   command:
+.. _building_from_source-centos8:
 
-   .. code-block:: console
+CentOS 8
+~~~~~~~~
 
-       cd ~/tarantool
-       $ git submodule update --init --recursive
+..  code-block:: console
 
-3. Use CMake to initiate the build.
+    $ yum install epel-release
 
-   .. code-block:: console
+    $ curl -s https://packagecloud.io/install/repositories/packpack/backports/script.rpm.sh | sudo bash
```

Comment:
I guess we need this on CentOS 7 (not CentOS 8) for python modules. See [here](https://github.com/tarantool/tarantool/pull/5751#discussion_r575500179). Or there is some other reason?